cluster/src/main/java/com/hartwig/pipeline/metadata/AddDatatypeToFile.java

```diff
@@ -21,4 +23,9 @@ public void apply(final SbpRestApi api, final AddFileApiResponse file) {
     public String path() {
         return path;
     }
+
+    @Override
+    public String toString() {
```

Comment:
How do these toStrings relate to the bug?

Reply:
I mentioned in the commit message but the detail is that as part of testing this change I made the pipeline temporarily fail on only one of the CRAM operations. It was useful to be able to see that the correct one was being registered at the end of the run (all the operations that are applied already get logged) and having a proper `toString` on the operations provided that.